content/gateway_guide.md

```diff
@@ -506,46 +524,37 @@ You may also enable `RequireAuth` for your cold wallet in order to use the [Auth
 
 If ACME decides to use Authorized Accounts, ACME creates an interface for users to get their Ripple trust lines authorized by ACME's cold account. After Alice has extended a trust line to ACME from her Ripple account, she goes through the interface on ACME's website to require ACME authorize her trust line. ACME confirms that it has validated Alice's identity information, and then sends a TrustSet transaction to authorize Alice's trust line.
```

Comment:
It's just a little disconcerting to see a paragraph about how ACME should generally use RequireAuth in a subsection of called "With Cold Wallets". It would make more sense to me to see it in the general introduction to the feature, whether or not that feature is being used as intended.